bin/workloads/micro/terasort/hadoop/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+current_dir=`dirname "$0"`
+root_dir=${current_dir}/../../../../..
+workload_config=${root_dir}/conf/workloads/micro/terasort.conf
+. "${root_dir}/bin/functions/load-bench-config.sh"
+
+enter_bench HadoopTerasort ${workload_config}
+show_bannar start
+
+rmr-hdfs $OUTPUT_HDFS || true
+
+SIZE=`dir_size $INPUT_HDFS`
+START_TIME=`timestamp`
+run-hadoop-job ${HADOOP_EXAMPLES_JAR} terasort \
+    -D ${REDUCER_CONFIG_NAME}=${NUM_REDS} \
```

Comment:
Can you update this to `-D mapreduce.job.reduces=${NUM_REDS}`? `REDUCER_CONFIG_NAME` will be removed later because `mapreduce.job.reduces` is the only value of it.